src/pages/en/guides/imports.md

```diff
@@ -135,6 +139,51 @@ It can also be useful to place images in the `public/` folder as explained on th
 Adding **alt text** to `<img>` tags is encouraged for accessibility! Don't forget to add an `alt="a helpful description"` attribute to your image elements. You can just leave the attribute empty if the image is purely decorative.
 :::
 
+## `Astro.glob()`
+
+[`Astro.glob()`](/en/reference/api-reference/#astroglob) is a way to import many files at once.
+
+`Astro.glob()` only takes one parameter: a relative [glob pattern](/en/guides/imports/#globpatterns) matching the local files you'd like to import. It’s asynchronous, and returns an array of each matching file's exports.
+
+```astro title="src/components/my-component.astro"
+---
+// imports all files that end with `.md` in `./src/pages/post/`
+const posts = await Astro.glob('../pages/post/*.md'); 
+---
+<!-- Renders an <article> for the first 5 blog posts -->
+<div>
+{posts.slice(0, 4).map((post) => (
+  <article>
+    <h1>{post.frontmatter.title}</h1>
+    <p>{post.frontmatter.description}</p>
+    <a href={post.frontmatter.url}>Read more</a>
+  </article>
+))}
+</div>
+```
+
```

Comment:
For `Astro.glob()` to work, the glob pattern must be a string literal. It can't contain any variables. See [the troubleshooting guide](https://docs.astro.build/en/guides/troubleshooting/#supported-values) for a workaround.